webrender/src/renderer.rs

```diff
@@ -414,6 +415,10 @@ pub struct Renderer {
 
     /// Map of external image IDs to native textures.
     external_images: HashMap<ExternalImageId, TextureId, BuildHasherDefault<FnvHasher>>,
+
+    // Optional trait object that allows to create a VRCompositor instance to handle WebVR commands.
+    // Some WebVR commands such as Vsync and SubmitFrame must be called in the WebGL render thread.
+    vr_compositor_creator: Arc<Mutex<Option<Box<VRCompositorCreator>>>>
```

Comment:
I only need access to the compositor creator from the render_backend. I could add an Option<Box<VRCompositorCreator>> to the RendererOptions struct if that's not a problem. I followed other setter patterns like set_render_notifier, set_render_notifier  and kept RendererOptions intact. VRCompositor creator is know at init time and won't change, so we can avoid the Arc/Mutex wrappers.

Let me know if you like the RenderOptions change and I'll update this PR.